### ISSUE TITLE
User Testing: test app without sign-in feature to focus on app interactivity

### ISSUE THREAD

**Issue by tesla809** [COVID-19-electronic-health-system/Corona-tracker#287]
## Summary
Let's test the app without the sign-in feature to focus on the app's interactivity.
The sign in flow has some issues based on the three user interviews I performed. 
It blocking the testing of the app's other features. 
On the suggestion of one user, we should have an instance of the app without the sign-in.
That way we can test the maps, swiping cards, etc.

## Motivation

Why are we doing this? 
The current blockstack log-in is causing issues with most users I am interviewing.
We can test the sign-in feature separately to gain insights on how to improve.
In the meanwhile, we can also test the features we are building without the sign-in to 
allow for faster iteration times on the other core features.

In short, we can test features in parallel.

What use cases does it support? 
Improving the other features of app with feedback from real users.

What is the expected outcome? 
We can gain user feedback related to the features and improve them iteratively

## Describe alternatives you've considered
Fixing the blockstack sign-in first.
Using blockstack/connect was suggested. But, this will require time that may block the front end team during its implementation. 

## Additional context
Nope.

**Comment by SomeMoosery** [comment]
I don't think we need this at this point - at least, I should scope out issues for creating true QA and Staging environments, and then perhaps in one of those we'll add a way to not require tests
